osxcollector/output_filters/summary_filters/html.py

```diff
@@ -327,4 +329,11 @@ def _print_val(self, val):
                 self._write('</li>')
             self._write('</ul>')
         elif isinstance(val, basestring) or isinstance(val, Number):
+            self._encode_val(val)
+
+    def _encode_val(self, val):
+        try:
             self._write(u'<span class="val">{0}</span>'.format(val))
+        except UnicodeDecodeError as err:
+            sys.stderr.write('Unicode decode error while processing the value: {0}\n{1}\n'.format(val, err))
+            self._write('<span class="val">{0}</span>'.format(val))
```

Comment:
Yeah, that would be ideal.

The only problem I am getting now is `UnicodeDecodeError` in the line:
```python
self._write(u'<span class="val">{0}</span>'.format(val))
```
[Here](https://gist.github.com/jjsendor/44b4d4c91a9854a3eb02481a5ab560de) is the whole method.